static/style.css

```diff
@@ -1,3 +1,11 @@
+:root {
+    --color-alert-note: #a5d5fe;
+    --color-alert-tip: #b4fa72;
+    --color-alert-important: #ff8ffd;
+    --color-alert-warning: #fefdc2;
+    --color-alert-caution: #ff8272;
+}
```

Comment:
How do we feel about color variables on `:root` like this?

Note how it makes setting the colors for light theme *far* less verbose than it would be without these.

Reply:
Yes, this is true. I like it a lot. But I would prefer if we could refactor and do this for everything then🙈 Would you mind looking into that? It would make the CSS much easier to read and use

Since we're already talking about refactoring CSS: We could also consider to start using something like https://lesscss.org or some other compiler for the CSS to make it even easier to work with. What do you think? If you like this or another one I can add it to the dev and build infrastructures

Reply:
Is it a lot of work to change all the css to this style?

Would it be worth letting this merge and the re-unifying the rest with a seperate PR?

I don't know enough about css compliers. Sometimes less is more, but maybe the benefits are enough to justify the additional complexity. Would it make it more confusing for users to then implement their custom css files?

Reply:
> Is it a lot of work to change all the css to this style?

Somewhat much but I think it'd be worth it for more unified colors & easier maintenance and adding new features

> Would it be worth letting this merge and the re-unifying the rest with a seperate PR?

Yes, definitely

> I don't know enough about css compliers. Sometimes less is more, but maybe the benefits are enough to justify the additional complexity. Would it make it more confusing for users to then implement their custom css files?

It would make the build infrastructure slightly more complex but the code would get a lot more readable. User-CSS would not be affected at all

Reply:
>> Would it be worth letting this merge and the re-unifying the rest with a seperate PR?
>
> Yes, definitely

+1 from me, I don't think there's necessarily a problem with having just these in `:root`

Reply:
> But I would prefer if we could refactor and do this for everything then🙈 Would you mind looking into that?

Yeah lets start an issue

Reply:
> Yeah lets start an issue

Nice, can you open it? Let's debate there if we also want to introduce a CSS compiler in the same context :)

Reply:
Alright: https://github.com/jannis-baum/Vivify/issues/120